### PR TITLE
Rickie redesign 3

### PR DESCRIPTION
**Pull request by rickiesherman**
1: Images are now blocks, so they never are inline.
screen shot 2017-01-24 at 5 54 35 pm
screen shot 2017-01-24 at 5 54 37 pm

2: Search input is bigger, works better in mobile mode
screen shot 2017-01-24 at 5 55 15 pm
screen shot 2017-01-24 at 5 55 22 pm

3: Adjusted height of sidebar in mobile width
screen shot 2017-01-24 at 5 57 06 pm
screen shot 2017-01-24 at 5 57 08 pm

4: Adds sharing functionality
screen shot 2017-01-24 at 5 58 34 pm

### CODE REVIEW

**Comment by rickiesherman** [comment]
Also: Images have a max-width of 640px and the article content has a max width of 880. That way, both are more legible.

**Comment by tosih** [comment]
For social, can we add Facebook?
For images, can we add some sort of lightbox implementation?

File: _sass/articles/article.scss [hunk 1]
```diff
@@ -143,7 +144,7 @@
   }
 
   // link
-  a:not(.well-anchor) {
+  a:not(.well-anchor, .share) {
```
Comment (taylordolan) [inline _sass/articles/article.scss]: This breaks the styling for normal `a` tags that don't have classes and are inside `.article` elements.

**Review by taylordolan — CHANGES_REQUESTED** [review]

File: _sass/articles/article.scss (hunk 1 above)
Comment (rickiesherman) [reply _sass/articles/article.scss]: I believe I got this taken care of, lmk

**Review by taylordolan — APPROVED** [review]

**Pull request merged**